src/test/java/com/csbank/v1/BankAccountTest.java

```diff
@@ -4,8 +4,7 @@
 import org.junit.runner.RunWith;
 import org.mockito.junit.MockitoJUnitRunner;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
I'll help you install codestyle in intelliJ.  But it's considered good coding practice to avoid importing .*,  import should be explicit on which class you're importing.